ui/app/templates/permission_scheduled_changes.html

```diff
@@ -0,0 +1,90 @@
+<div loader ng-show="loading"></div>
+
+<h2>
+  <span ng-show="!scheduled_changes_permissions_count">Scheduled Permissions Changes</span>
+  <span ng-show="scheduled_changes_permissions_count">History for Scheduled Permissions Change</span>
```

Comment:
Is this used? I don't see references to scheduled_changes_permissions_count in the controller.